### DISCUSSION TITLE
I accidentally edited the beatmap for a certain difficulty, how do I restore it to its original state?

### DISCUSSION THREAD

**Discussion by LianSheng197** [ppy/osu#25091] [Q&A]
Just now I was checking the arrangement of notes in Edit mode, and when I left, I clicked Save by mistake, and now it's in Local mode.

Since there are a lot of offline scores for beatmaps of other difficulty levels (I like to keep the offline scores, they are my growth records), I don't want to delete them, so is there a way to recover just the online status of this beatmap, instead of deleting the whole set of beatmaps and re-downloading them again?

![image](https://github.com/ppy/osu/assets/18535495/8ae36aaf-f261-430e-8470-6190026044f9)

**Comment by nagi-desuuu** [comment]
Can't you just use the update button that's on the beatmap card?

**Reply by LianSheng197** [reply]
I don't see the Update button...
Maybe I should restart the game.

**Reply by LianSheng197** [reply]
I reopened osu and still don't see the Update button.
If there is a button, it should appear in the red box, but I don't see it.

![image](https://github.com/ppy/osu/assets/18535495/019d4660-4bea-4d7d-b63c-139be04a2f3e)

**Reply by peppy** [reply]
Unfortunately the update flow has been broken for reverting local changes for a while now (see https://github.com/ppy/osu/issues/23524)...

Your best solution is to delete the beatmap and download it again using the beatmap listing.
